backend/tests/unit/controllers/posts-controller.test.js

```diff
@@ -113,33 +109,21 @@ describe('createPostHandler', () => {
 
     await createPostHandler(req, res);
 
-    expect(res.status).toHaveBeenCalledWith(400);
+    expect(res.status).toHaveBeenCalledWith(500);
     expect(res.json).toHaveBeenCalledWith({ message: errorMessage });
   });
 });
 
 describe('getAllPostsHandler', () => {
   it('should get all posts', async () => {
-    const req = {};
```

Comment:
Can we try to get consistency in the description like :

get all posts - success 
get all posts -failure 

OR

[POSITIVE] - get all posts 
[NEGATIVE ]- get all posts

Feel free to come up with new version of your self, or something like above we can stick on to, so that when we see in the console, we exactly know what is happening :)

Reply:
Thank you @hemant14050, can you once confirm, if you wanna take this up as a refactoring issue and raise new PR ? We can merge this, This already is very useful and serves the very purpose of having the test cases for our application

Reply:
Hey @krishnaacharyaa, I'll take up the task of refactoring and complete it soon. So, I think no need for new PR.

Reply:
Glad to hear @hemant14050, Thank you for confirming :)